packages/wrangler/src/__tests__/publish.test.ts

```diff
@@ -4717,9 +4725,12 @@ addEventListener('fetch', event => {});`
 			- Text Blobs:
 			  - TEXT_BLOB_ONE: my-entire-app-depends-on-this.cfg
 			  - TEXT_BLOB_TWO: the-entirety-of-human-knowledge.txt
-			- Unsafe:
+			- Unsafe Bindings:
```

Comment:
I can't decide whether this log message would count as a breaking change...
It is feasible that someone is looking for `Unsafe:` and not `Unsafe Bindings:`.

Reply:
It might be safer to just move the new section (Unsafe Metadata) further down after the Vars, etc; and leave this bit alone?

Reply:
Done.  Moved the output for unsafe.metadata to the end of the list.